Northstar.Custom/mod/scripts/vscripts/gamemodes/cl_gamemode_fastball.gnut

```diff
@@ -92,8 +92,9 @@ void function FastballRespawnPlayerEffects_Threaded()
 	// sometimes this seems to get called before the player has respawned clientside, so we just wait until the client thinks they're alive
 	entity player = GetLocalClientPlayer()
 	
-	while ( !IsAlive( player ) || !IsValid( player.GetCockpit() ) )
+	while ( !( IsAlive( player ) && IsValid( player.GetCockpit() ) )
 		WaitFrame()
 		
-	StartParticleEffectOnEntity( player.GetCockpit(), GetParticleSystemIndex( $"P_pod_screen_lasers_OUT" ), FX_PATTACH_ABSORIGIN_FOLLOW, -1 )
+	if ( IsValid( player.GetCockpit() ) 
```

Comment:
This is just moving where you do the IsValid check, there's no reason this shouldn't have the same issues as the previous pr
please ensure that the logs provided were from a client with the previous fix installed